firmware-nucleo/smart-air-hockey-table/App/Inc/driver_halleffect.h

```diff
@@ -3,8 +3,8 @@
 
 #include <inttypes.h>
 
-extern volatile uint32_t halleffect_cols;
-extern volatile uint16_t halleffect_rows;
+extern uint32_t halleffect_cols;
+extern uint16_t halleffect_rows;

```

Comment:
these should probably stay volatile since any code block reading this `extern` variable may be interrupted.

Reply:
The only code that reads or writes those values is contained within the GameTick control flow, so there's no worry about other interrupts here.

Reply:
I know, but we don't know how it'd be handled in the future. I'd rather we not get stuck by this as a bug in the future.